src/spring-cloud/azext_spring_cloud/vendored_sdks/appplatform/_app_platform_management_client.py

```diff
@@ -23,7 +23,6 @@
     from typing import Any, Optional
 
     from azure.core.credentials import TokenCredential
-    from azure.core.pipeline.transport import HttpRequest, HttpResponse
```

Comment:
This change is irrelavant with 2022-03-01-preview api version,but is done by SDK generated by swagger PR. And manually checked the code, these imports are not used.
Same for the refactoring in line 74~79, introduced by sdk from swagger PR.